app/components/AppScene.brs

```diff
@@ -18,14 +24,18 @@ end function
 
 function init() as Void
     launchDarklyNode = m.top.findNode("launchDarkly")
-
-    config = LaunchDarklyConfig("YOUR MOBILE KEY HERE", launchDarklyNode)
+    
+    config = LaunchDarklyConfig(mobileKey, launchDarklyNode)
     config.setLogLevel(LaunchDarklyLogLevels().debug)
-
-    user = LaunchDarklyUser("user-key")
+  
+    // Set up the user properties. This user should appear on your LaunchDarkly users dashboard
+    // soon after you run the demo.
+    user = LaunchDarklyUser("example-user-key")
+    user.setName("Sandy")
 
     LaunchDarklySGInit(config, user)
-
+    // TKTK initialization success/failure messages needed.
```

Comment:
Ideally, we'd add initialization success/failure messages here per step 3: https://launchdarkly.atlassian.net/wiki/spaces/ENG/pages/1499562073/SDK+hello+world#Example-README.md

Reply:
I'm the closest thing we have. I don't know off the top of my head how (or whether) this works in Roku, but I'll take a look.